### PR TITLE
Merge kilo into master

### PR DESCRIPTION
**Pull request by pjbreaux**
A new release (v1.0.1) was just generated, and now we will merge the kilo branch into master.

### CODE REVIEW

**Comment by pjbreaux** [comment]
Hey @zancas, I'd like your assistance in fixing these conflicts. Thanks!

File: .travis.yml [hunk 1]
```diff
@@ -18,11 +16,24 @@ install:
...
+        - secure: SUKOH8gDCnSCO969XPttA9bxTI6BACqTJgCLl6MDY1siRhTOyVmww1g989Pg9uKc1Hy+IKGDFdy0/+SXURCbj9xMEVOCGGhdM6Jo2xa/6yjTsS1X4auxegBaGEdSWVD7zUz8qKc/uc3Z0L3K5AHG4zHyJQeFsCcxouMktwCsreYap88H7weCQItEXxb9aRFOMZHeRsaJ8RLfhmF3hHHgQtase4JMbXM8uAm9tgOH9cCq0+W3/eaAGFFgzKH1D7GoWG5a94kNHFRgrFP6V560r6chxu7m3CL4EiPuAfExAQz3nlb7sCgWzWqNqsa9GbCi6GiAXS0OnQ10BPY/4oPWdWvi4EeX1RWeEfZmazo2XvSoVIdG6JHawH4DhHeReWY/hy0wLNGjeUxX29kJ6OLWvSm3zw+j1Gjs5TWqS85Aa3DY4bbb0kWfyW2EjmPTuZc91FUIYPM8W4DnXF+qqB46dbTZfE0aHJ/2sEz12ZaJxO4Uvym70/yYW77rfBBHg7o1ZCqTEFQXIx0+sxhHwbEZBtXt0lmPLR2vs45Yn5b3mX9i3gob8PVJ/ytE9UR+cjKWM665J2PhWo0rkW+J7+s4CZCJwQ8Noi6Rj8Ex3ImXvpdyB2UiYP5xMPDbzbPt6yO3GperXkMwjk+dixH+LA2f1Z2FlRC2oGAeue2oo3ou9X8=
       on_success: change
```
Comment (zancas) [inline .travis.yml:38]: @pjbreaux:  Are these "secure" strings supposed to be published?

File: f5_heat/resources/common/mixins.py [hunk 2]
```diff
@@ -14,6 +14,21 @@
...
+    return func_wrapper
+
+
+def f5_bigip(func):
+    def func_wrapper(self, *args, **kwargs):
+        self.get_bigip()
+        func(self, *args, **kwargs)
+    return func_wrapper
+
+
```
Comment (zancas) [inline f5_heat/resources/common/mixins.py:31]: I infer by the presence of the "self" param that these are "method_wrapper" 's not "func_wrapper" 's.

File: f5_heat/resources/common/mixins.py [hunk 3]
```diff
@@ -14,6 +14,21 @@
...
+        self.get_bigip()
+        self.set_partition_name()
+        return func(self, *args, **kwargs)
+    return func_wrapper
+
+
+def f5_bigip(func):
+    def func_wrapper(self, *args, **kwargs):
+        self.get_bigip()
+        func(self, *args, **kwargs)
```
Comment (zancas) [inline f5_heat/resources/common/mixins.py:28]: Is the inner function really not supposed to be returned?

File: requirements.docs.txt [hunk 4]
```diff
@@ -1,8 +1,3 @@
-cryptography
-eventlet==0.17.4
-funcsigs
-netaddr==0.7.18
-pyopenssl==0.15.1
-requests
+f5-sdk>=0.1.1a2
```
Comment (zancas) [inline requirements.docs.txt:1]: This should probably be removed, or updated to 0.1.2

File: requirements.txt [hunk 5]
```diff
@@ -1,7 +1 @@
-cryptography
-eventlet==0.17.4
-funcsigs
-netaddr==0.7.18
-pyopenssl==0.15.1
-requests
-suds==0.4
+f5-sdk>=0.1.2
```
Comment (zancas) [inline requirements.txt]: This isn't correct.

Comment (pjbreaux) [inline requirements.txt:1]: What isn't?

File: .travis.yml (hunk 1 above)
Comment (pjbreaux) [inline .travis.yml:38]: Hmm don't know. They have been since the beginning.

File: requirements.txt (hunk 5 above)
Comment (zancas) [inline requirements.txt]: Oops...  I'm wrong.

**Pull request merged**